ControlRoomApplication/ControlRoomApplication/Controllers/ControlRoomController/ControlRoomController.cs

```diff
@@ -13,160 +13,10 @@ public class ControlRoomController
         public ControlRoom ControlRoom { get; set; }
         private static readonly log4net.ILog logger =
             log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
-        private Thread WeatherMonitoringThread;
-        private Thread SensorStatusUpdateThread;
-        private bool KeepWeatherMonitoringThreadAlive;
-        private bool KeepSensorStatusUpdateThreadAlive;
-
-        // Weather Station override
-        //public bool weatherStationOverride = false;
 
         public ControlRoomController(ControlRoom controlRoom)
         {
             ControlRoom = controlRoom;
-            WeatherMonitoringThread = new Thread(new ThreadStart(WeatherMonitoringRoutine)) { Name = "Weather Monitoring Routine" };
-            KeepWeatherMonitoringThreadAlive = false;
-        }
-
-        public bool StartWeatherMonitoringRoutine()
```

Comment:
Make sure you get rid of everything that uses these methods too. Mainly in the Main form.

- [ ] Remove all calls to deleted methods